src/uwp/Host/UWPHost/HostEnvironment.cpp

```diff
@@ -10,14 +10,9 @@
 //
 
 #include "windows.h"
+#include "appmodel.h"
 #include "HostEnvironment.h"
 
-// This should ideally come from WINDOWS SDK header but is missing in earlier versions
-#ifndef PACKAGE_FILTER_IS_IN_RELATED_SET 
-#define PACKAGE_FILTER_IS_IN_RELATED_SET 0x40000
```

Comment:
I think you may need to keep this. The CI build machine does not seem to have Windows SDK with it.